tests/test_backend_api.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from grid2op.tests.aaa_test_backend_interface import AAATestBackendAPI
+
+from PandaModelsBackend import PandaModelsBackend
+
+class TestBackendAPI_PandaModelsBackend(AAATestBackendAPI, unittest.TestCase):
+    def make_backend(self, detailed_infos_for_cascading_failures=False):
+        return PandaModelsBackend(detailed_infos_for_cascading_failures=detailed_infos_for_cascading_failures)
+    
+    # The following tests are skipped as these tests may not relevant for the PandaModelsBackend backend
+    # in the PandaModelsBackend as PandaPower runnpp doesn't converge but runpm does
+    def test_17_isolated_gen_stops_computation(self):
```

Comment:
would `@unittest.expectedFailure` be relevant? https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures it says it's only for failures in the assert, not for failures at RT, so maybe not.